packages/expo-av/src/Audio/Recording.types.ts

```diff
@@ -1,48 +1,215 @@
-// TODO: For consistency with PlaybackStatus, should we include progressUpdateIntervalMillis here as
-// well?
+import { Recording } from './Recording';
+import {
+  AndroidAudioEncoder,
+  AndroidOutputFormat,
+  IOSAudioQuality,
+  IOSOutputFormat,
+} from './RecordingConstants';
+
+// TODO: For consistency with PlaybackStatus, should we include progressUpdateIntervalMillis here as well?
```

Comment:
It's not my comment, it was in there, but I can check this. 😅 

My general idea is to convert AV next (Audion and Video will be done), then try to clean up the code since all tree of the APIs are inside the same package source.